cli/data/get.go

```diff
@@ -52,31 +53,38 @@ var getCmd = &cobra.Command{
 Example:
 
 $ paddle data get -b experimental --bucket roo-pipeline trained-model/version1 dest/path
+$ paddle data get -b experimental --bucket roo-pipeline --files file1.csv,file2.csv trained-model/version1 dest/path
 `,
 	Run: func(cmd *cobra.Command, args []string) {
+		var files []string
+
 		if getBucket == "" {
 			getBucket = viper.GetString("bucket")
 		}
 		if getBucket == "" {
 			exitErrorf("Bucket not defined. Please define 'bucket' in your config file.")
 		}
+		if getFiles != "" {
+			files = strings.Split(getFiles, ",")
+		}
 
 		source := S3Path{
 			bucket: getBucket,
 			path:   fmt.Sprintf("%s/%s/%s", args[0], getBranch, getCommitPath),
 		}
 
-		copyPathToDestination(source, args[1])
+		copyPathToDestination(source, args[1], files)
 	},
 }
 
 func init() {
 	getCmd.Flags().StringVarP(&getBranch, "branch", "b", "master", "Branch to work on")
 	getCmd.Flags().StringVar(&getBucket, "bucket", "", "Bucket to use")
 	getCmd.Flags().StringVarP(&getCommitPath, "path", "p", "HEAD", "Path to fetch (instead of HEAD)")
+	getCmd.Flags().StringVarP(&getFiles, "files", "f", "", "A list of files to download separated by comma")
```

Comment:
If you make this a `StringSliceVarP`, it does the splitting for you (and allows you alternatively to repeat the flag several times)

Reply:
That's really cool, the code is much cleaner now :)